dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -103,6 +103,7 @@
     private static final String QUAY_URL = "https://quay.io/api/v1/";
     private static final String BITBUCKET_URL = "https://bitbucket.org/";
     private static final String GITLAB_URL = "https://gitlab.com/";
+    private static final String ZENODO_URL = "https://sandbox.zenodo.org/";
```

Comment:
This should not be hard-coded in here. This should be configurable, otherwise we will need to make a new build to run against the non-sandbox Zenodo.

Reply:
Have to figure out how to do this...

Reply:
done